package.json

```diff
@@ -54,7 +56,7 @@
     "@types/xml2js": "^0.4.5",
     "cross-env": "^7.0.2",
     "deepmerge": "^4.2.2",
-    "detox": "^18.16.0",
+    "detox": "^18.19.0",
```

Comment:
If you update dependencies, a package-lock should also be in this PR

Reply:
Alright! Committed this time 👍